src/typings/global.ts

```diff
@@ -124,6 +124,7 @@ declare global {
     attributes: BiggySKUAttribute[]
     sellers: BiggySeller[]
     policies: BiggyPolicy[]
+    catalogAttributes: BiggySKUCatalogAttribute[]
```

Comment:
```suggestion
    catalogAttributes?: BiggySKUCatalogAttribute[]
```
just to validate when the property doesn't exist in SKU object